Dockerfile

```diff
@@ -39,8 +39,15 @@ COPY --from=cargo-build /driver /usr/local/bin/driver
 ENTRYPOINT [ "driver" ]
 
 FROM intermediate as orderbook
+RUN apt-get update && \
+    apt-get install -y build-essential cmake git zlib1g-dev libelf-dev libdw-dev libboost-dev libboost-iostreams-dev libboost-program-options-dev libboost-system-dev libboost-filesystem-dev libunwind-dev libzstd-dev git
+RUN git clone https://invent.kde.org/sdk/heaptrack.git /heaptrack && \
+    mkdir /heaptrack/build && cd /heaptrack/build && \
+    cmake -DCMAKE_BUILD_TYPE=Release -DBUILD_GUI=OFF .. && \
+    make -j$(nproc) && \
+    make install
 COPY --from=cargo-build /orderbook /usr/local/bin/orderbook
-ENTRYPOINT [ "orderbook" ]
+ENTRYPOINT ["/bin/sh", "-c", "exec heaptrack -o /tmp/heaptrack/heaptrack.orderbook.$(shuf -i 1-99999 -n 1).gz orderbook"]
```

Comment:
Is it necessary that the `/tmp` path gets defined here and in the shell script below?

Reply:
Yes, because this is where the [PVC is defined](https://github.com/cowprotocol/infrastructure/pull/1555).